src/Symfony/Bundle/SecurityBundle/Tests/Functional/app/AppKernel.php

```diff
@@ -63,6 +63,8 @@ public function __construct($testCase, $rootConfig, $environment, $debug)
         $this->rootConfig = $rootConfig;
 
         parent::__construct($environment, $debug);
+
+        $this->name = $this->getName().md5($this->rootConfig);
```

Comment:
IMO it's cleaner to overwrite the `getName` method. The problem is that `name` is protected when it should actually be private which is a design flaw. There should not be two extension point for the same thing (public method & protected property).